robot_skills/src/robot_skills/util/entity.py

```diff
@@ -146,7 +146,7 @@ def from_entity_info(e):
     super_types = e.types
 
     # TODO: this must be part of the definition of the entity in ED.
-    if e.has_shape and "amigo" not in e.id and "sergio" not in e.id and e.id is not "floor" and e.id is not "walls":
+    if e.has_shape and "amigo" not in e.id and "sergio" not in e.id and e.id != "floor" and e.id != "walls":
```

Comment:
On rwc2017 we had wall_240 and other lengths, so better to ```"wall" not in e.id```. But best is to move it to ED.

Reply:
@Rayman will fix this